src/Serilog.Sinks.Console/Sinks/SystemConsole/Formatting/ThemedJsonValueFormatter.cs

```diff
@@ -154,7 +154,7 @@ protected override int VisitDictionaryValue(ThemedValueFormatterState state, Dic
                         : ConsoleThemeStyle.Scalar;
 
                 using (ApplyStyle(state.Output, style, ref count))
-                    JsonValueFormatter.WriteQuotedJsonString((element.Key.Value ?? "null").ToString(), state.Output);
+                    JsonValueFormatter.WriteQuotedJsonString((element.Key.Value?.ToString() ?? "null"), state.Output);
```

Comment:
object.ToString() can return null. This change will deal with this uncommon case.

Reply:
This could end up being slightly misleading; in the original version, `null` indicates that the value itself is `null`; when the value is non-null, but returns `null` from `ToString()`, it might pay to use the empty string instead?

Reply:
I commented about this on the other comment.